pages/issuer/issue.vue

```diff
@@ -76,10 +78,14 @@
         step.value = Step.DONE;
       }, 2000)
     })
-    addToLog("[Issuer] Credential sent successfully!")
+    addToLog("Credential received successfully!", "Wallet", "Issuer")
   }
 
-  function addToLog(logMessage: string) {
-    logMessages.value.push(logMessage)
+  function addToLog(message: string, source: string = "Issuer", target: string | null = "Wallet") {
```

Comment:
You never use the default value - at least not in the lines changed in this PR. So I propose to remove the default value and mark `target` as optional. Then you can remove all those `null`. But you'll have to test against `undefined` then.

```suggestion
  function addToLog(message: string, source: string = "Issuer", target?: string) {
```